impl/src/main/java/io/jsonwebtoken/impl/io/RuntimeClasspathDeserializerLocator.java

```diff
@@ -34,6 +34,8 @@
             return Classes.newInstance("io.jsonwebtoken.io.JacksonDeserializer");
         } else if (isAvailable("io.jsonwebtoken.io.OrgJsonDeserializer")) {
             return Classes.newInstance("io.jsonwebtoken.io.OrgJsonDeserializer");
+        } else if (isAvailable("com.google.gson.GsonBuilder")) {
+            return Classes.newInstance("io.jsonwebtoken.io.GsonDeSerializer");
```

Comment:
This should be "io.jsonwebtoken.io.GsonDeserializer" not "io.jsonwebtoken.io.GsonDeSerializer"

Reply:
Ok patching.

Reply:
Missed DeSerializer -> Deserializer on both this and the line above now.